src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPageFairObject.java

```diff
@@ -0,0 +1,46 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
+
+import com.wikia.webdriver.common.core.networktrafficinterceptor.NetworkTrafficInterceptor;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import org.junit.Assert;
+import org.openqa.selenium.*;
+
+public class AdsPageFairObject extends AdsBaseObject {
+    private static final String PATTERN_PAGEFAIR = "http://stats.pagefair.com/stats/page_view_event/[0-9A-Z]{16}/" +
```

Comment:
You can move whole string to new line. Like here: https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetection.java#L13